tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -39,8 +36,10 @@
 NUM_USERS_TO_CREATE = 5
 NUM_USERS_TO_TEST = 3
 
-ADMIN_PASSWORD = "".join(random.choices(string.ascii_letters + string.digits, k=60))
-USER_PASSWORD = "".join(random.choices(string.ascii_letters + string.digits, k=60))
+# ADMIN_PASSWORD = "".join(random.choices(string.ascii_letters + string.digits, k=60))
+# USER_PASSWORD = "".join(random.choices(string.ascii_letters + string.digits, k=60))
+ADMIN_PASSWORD = "TestPass123"
```

Comment:
We can keep random password. To do that, we must retrieve the password secret from the stack output and read the password at runtime